### PR TITLE
Lp 40 remove Pokémon functionality 

### LINKED ISSUE #40: Add remove Pokémon functionality
**Is your feature request related to a problem? Please describe.**
There is no way to remove two Pokémon searched to one Pokémon searched

**Describe the solution you'd like**
Add remove Pokémon functionality to the search component

### PR DESCRIPTION
**Pull request by Lap343**
## Changes
_List Changes Introduced by this PR_
1. added a remove last Pokémon button
2. refactored code 
3. added functionality that removes the last Pokémon searched and re-renders the screen

## Purpose
To be able to remove the last Pokémon searched if it wasn't the one you were looking for or wanted to look for a new one

## Approach
refactoring the code so the left button acts as the remove Pokémon button and the other buttons acts as toggles 

Closes #40

### CODE REVIEW

**Review by BlackCubes — APPROVED** [review]
Dis gun be gewd!

**Review by Wolftownn — APPROVED** [review]
Looks great Luis!

File: pokemon-app/src/components/Search.js [hunk 1]
```diff
@@ -13,65 +13,70 @@ const Search = ({ getPokeByNameOrIdAPI, isEnemy, setIsEnemy }) => {
...
     e.preventDefault();
-
     const pokeLowerCase = pokemonSearchValue.toLowerCase();
 
     setHasSelected(true);
     getPokeByNameOrIdAPI(pokeLowerCase);
   };
+  const removePokemon = () => {
+    if(enemyPokemon === null){
+      if(pokemon !== null){
```
Comment (kmausolf) [inline pokemon-app/src/components/Search.js:23]: You can also shorthand this as `if(!enemyPokemon)` and `if(pokemon)`, as `null` is falsy.

File: pokemon-app/src/components/Search.js [hunk 2]
```diff
@@ -13,65 +13,70 @@ const Search = ({ getPokeByNameOrIdAPI, isEnemy, setIsEnemy }) => {
...
           value={pokemonSearchValue}
           onChange={onChange}
         />
-      </form>
 
-      <div className="twosearch-pokemon">
-        <div className="twosearch-pokemon__title">
-          <p className="paragraph-versus">Versus</p>
-        </div>
+        <button type="submit" className="search-button">Search</button>
```
Comment (kmausolf) [inline pokemon-app/src/components/Search.js:46]: I notice that you're mixing controlled form logic with uncontrolled form logic.  Which one are you trying to implement?

**Pull request merged**